docs/service-mesh/03-02-istio-patch.md

```diff
@@ -3,29 +3,28 @@ title: Istio patch
 type: Details
 ---
 
-As a core component, Istio installs with every Kyma deployment by default. The installation consists of two steps:
+As a core component, Istio is installed with every Kyma Deployment by default. To run Kyma, this default Istio installation needs some changes. The [Istio Kyma patch](../../components/istio-kyma-patch/README.md) application patches the already existing Istio installation so that Kyma can use it. Considering the above, the installation includes the following steps:
 
-1. Istio installs using the official charts from the currently supported release. The charts that are currently used are stored in the `resources/istio` directory. The installation is customized as described in [this document](#details-istio-customization)
+1. Istio is installed using the official charts from the currently supported release. The charts reside in the `resources/istio` directory.
 
-2. A custom Istio patch job runs and checks if the detected Istio deployment meets the following criteria:
-  - A specific version of Istio is installed. The required version is defined in the [`values` file](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/values.yaml#L11) of the patch.
+2. A custom Istio patch job runs and checks if the detected Istio Deployment meets the following criteria:
+  - A specific version of Istio is installed. The required version resides in the [`values`](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/values.yaml#L11) file.
   - Mutual TLS (mTLS) policy is enabled and set to `strict`.
   - [Istio policy enforcement](https://istio.io/docs/tasks/policy-enforcement/enabling-policy/) is enabled. 
   - Automatic sidecar injection is enabled.
   - Istio `policies.authentication.istio.io` CustomResourceDefinition (CRD) is present in the system.
 
-If the Istio deployment fails to meet any of these criteria, the patch fails, which results in a failed installation. If the installation failed, get the Istio patch logs to find out which criteria the Istio deployment didn't' meet. Run: 
+If the Istio Deployment fails to meet any of these criteria, the patch fails, which results in a failed installation. In such a case, get the Istio patch logs to find out which criteria the Istio Deployment didn't meet. Run: 
```

Comment:
it's not a "deployment" in a Kubernetes resource-kind-of-way, it's just a generic term